internal/helpers/helpers.go

```diff
@@ -44,3 +53,17 @@ func GetAftermarketDeviceImageUrl(baseURL string, tokenID int) string {
 func GetVehicleDataURI(baseURL string, tokenID int) string {
 	return fmt.Sprintf("%s%d", baseURL, tokenID)
 }
+
+func SlugString(term string) string {
```

Comment:
What does this function do? It might be worth adding a comment and some tests. The meat seems to be line 62 but I can't tell offhand what it does.

Reply:
Maybe some tests would be good too.

Reply:
It looks like there's a version of this in shared?